src/garage/experiment/experiment.py

```diff
@@ -701,22 +701,28 @@ def make_launcher_archive(*, git_root_path, log_dir):
         log_dir (str): Absolute path to the log directory.
 
     """
-    files_to_archive = subprocess.check_output(
+    git_files = subprocess.check_output(
         ('git', 'ls-files', '--others', '--exclude-standard', '--cached',
          '-z'),
         cwd=git_root_path).strip()
-    repo_size = sum([
-        os.stat(os.path.join(git_root_path, f)).st_size
-        for f in files_to_archive.split(b'\0')
-    ])
+    repo_size = 0
```

Comment:
This stuff should actually be in a separate PR. It fixes `wrap_experiment` when there are deleted files in the current launch repo.

Reply:
I have had this issue before. Very necessary